api/kessel/health/v1/health.proto

```diff
@@ -1,34 +1,34 @@
 syntax = "proto3";
 
-package kessel.relations.v1;
+package kessel.health.v1;
```

Comment:
Is there a reason this package changed from the other relation API protos?

Reply:
Looking at this again, no it could just be `kessel.relations.v1`. But I think my reasoning was that it was originally in a `health/v1` directory and so the linter complained and moving it into kessel/ while keeping the same structure and updating the package seemed reasonable.